app/javascript/mastodon/components/media_gallery.js

```diff
@@ -283,8 +283,9 @@ export default class MediaGallery extends React.PureComponent {
       if (width) {
         style.height = width / this.props.media.getIn([0, 'meta', 'small', 'aspect']);
       }
+    } else if (width) {
+      style.height = Math.floor(width / (16/9));
```

Comment:
Why `floor`?

Reply:
Do floating point pixels matter in height?

Reply:
I don't think it practically matters, even if they get rendered. But I don't think `floor` makes difference, either.